Packages/MIES/MIES_DAEphys_Macro.ipf

```diff
@@ -3616,6 +3602,9 @@ Window DA_Ephys() : Panel
 	GroupBox group_acq_tp,userdata(tabnum)=  "5",userdata(tabcontrol)=  "ADC"
 	GroupBox group_acq_daq,pos={31.00,554.00},size={208.00,62.00},disable=1,title="Data Acquisition"
 	GroupBox group_acq_daq,userdata(tabnum)=  "5",userdata(tabcontrol)=  "ADC"
+	Button button_hardware_rescan,pos={40.00,75.00},size={37.00,36.00},proc=ButtonProc_Hardware_rescan,title="\\Z20🔄"
```

Comment:
@t-b try this for the title "\\Z24\\F'Wingdings 3'P"

Reply:
I don't have that font. I'll make three proc pictures of mine. Also disable the button after locking.

Reply:
Done.